pilot/pkg/proxy/envoy/v1/authentication.go

```diff
@@ -46,7 +39,25 @@ func buildJwtFilter(policy *authn.Policy) *HTTPFilter {
 	}
 	return &HTTPFilter{
 		Type:   decoder,
-		Name:   jwtFilterName,
+		Name:   authn_plugin.JwtFilterName,
+		Config: config,
+	}
+}
+
+// buildAuthnFilter returns a authN filter for the policy.
+func buildAuthnFilter(policy *authn.Policy) *HTTPFilter {
```

Comment:
Atm we don't want to change the v1 code at all. All new features are going in to v2 only. This code should live in v2 and v1 should remain unchanged (cc @costinm).

Reply:
For the record, the filter will be hooked up with v2 listener via plugin. I want to wait for some plugin refactor (e.g [this](https://github.com/istio/istio/pull/4718)) before doing so.